### ISSUE TITLE
lambda on test

### ISSUE THREAD

**Issue by Riko0** [wy1iu/sphereface#126]
Does lamda in test have meaning?
usually its bigger than on train, but i guess it have no information in it, am i right?

I0906 19:50:07.181219 41380 solver.cpp:358] Iteration 2700, Testing net (#0)
I0906 19:50:07.997248 41380 solver.cpp:425]     Test net output #0: accuracy/top-1 = 0.93375
I0906 19:50:07.997248 41380 solver.cpp:425]     Test net output #1: lambda = 3.30186
I0906 19:50:07.998245 41380 solver.cpp:425]     Test net output #2: softmax_loss = 0.216583 (* 1 = 0.216583 loss)
I0906 19:50:08.126904 41380 solver.cpp:243] Iteration 2700, loss = 0.465629
I0906 19:50:08.126904 41380 solver.cpp:259]     Train net output #0: lambda = 3.08261
I0906 19:50:08.127900 41380 solver.cpp:259]     Train net output #1: softmax_loss = 0.465629 (* 1 = 0.465629 loss)
